calblueprint/vouchers4veggies
# [wip] Add hifi login screen

# ✨ New in this PR ✨

## One liner: what did you do? 🚨
Updated login screen to hifi version as per Figma design.

## How can the reviewer test your code? 👩‍💻
Run `npx expo start` and examine the login page. Check that the input fields react properly when selected.
Comment out the login screen and examine the start page.

## Any bugs you encountered or still having trouble with? 🐛
Styling was done based on percentages rather than fixed pixels; as such, the keyboard should never cover the inputs. If this may be an issue on other devices though, it might be good to add in adjustable inputs.
Input does not do any form of validation check yet.


## Resources 📔
Building upon the screens found in #2 

🥦 CC: @oahnh
